test/src/scheme/tendermint-int.json

```diff
@@ -11,37 +11,37 @@
                 "timeoutPrecommitDelta": 1000,
                 "timeoutCommit": 1000,
                 "genesisStakes": {
-                    "tccq8t6d5nxsd7pckgnswusmq6sdzu76kxa808t6m3gtygltrjqeeqncfggwh3": { "stake": 70000,
```

Comment:
It seems that this change is reverted in the merge commit.

Reply:
I modified it. Could you check it again please?